docs/reference/worker-configuration-reference.md

```diff
@@ -352,7 +352,7 @@ Example:
 ```yaml
 sources:
   local:
-    hooksEnabled: true  # true by default
+    hooksEnabled: false  # disabled by default
```

Comment:
We could document the default like the other options, i.e. add a line  `Default: false` above